### DISCUSSION TITLE
User selectable ADIF fields

### DISCUSSION THREAD

**Discussion by ea4k** [ea4k/klog#262] [Ideas]
I am starting to work on the [Issue-4](https://github.com/ea4k/klog/issues/4) that will add some user selectable [ADIF](http://www.adif.org/312/ADIF_312.htm) values to KLog.

It is quite close to impossible to show ALL the ADIF fields in the UI so I am adding two additional ADIF types:
One will be in the My Data tab and the other will be in the Other tab.

For example:
In the My Data tab I will include a combobox with the "My SOTA Ref" and in the Other tab I will include a Combobox with the "SOTA ref" so SOTA operators shall be able to manage SOTA info when they are in in the summit or at home, working other SOTA stations.

I am planning to add no more than 2-3 in the My Data and 2-3 in Other, to test the idea.
If it works, I will add more, if needed.

I am writing here to ask for your feedback.
Which are the ADIF types that you are missing most in KLog?

**Comment by creativetux** [comment]
sig/mysig/siginfo/mysiginfo for pota/wwff activities

**Reply by ea4k** [reply]
I will add it soon.
Thank you.
https://github.com/ea4k/klog/issues/508
